src/doc/DocAnnotator.js

```diff
@@ -545,7 +546,7 @@ class DocAnnotator extends Annotator {
         }
 
         this.selectionEndTimeout = setTimeout(() => {
-            if (this.createHighlightDialog) {
```

Comment:
Is the identity check no longer required? Is this property always defined?

Reply:
Adding the safety check back